rpc/api.proto

```diff
@@ -381,6 +382,26 @@ message FundTransactionResponse {
 	bytes change_pk_script = 3;
 }
 
+message UnspentOutputsRequest {
+	uint32 account = 1;
+	int64 target_amount = 2;
+	int32 required_confirmations = 3;
+	bool include_immature_coinbases = 4;
+}
+message UnspentOutputResponse {
+	message PreviousOutput {
```

Comment:
we can remove this message and inline all of these fields in the response type.  It was only a different message type before so it could be `repeated`.